Docker/start.sh

```diff
@@ -26,7 +26,7 @@ done
 
 export ADAGUC_PATH=/adaguc/adaguc-server-master/ && \
 export ADAGUC_TMP=/tmp && \
-/adaguc/adaguc-server-master/bin/adagucserver --updatedb \
+env -u ADAGUC_FORK /adaguc/adaguc-server-master/bin/adagucserver --updatedb \
```

Comment:
We need to discuss if it would be  a good solution to start this executable once in the python code. The reasoning is that the python code has logic to calculate the environment variables like `ADAGUC_ONLINERESOURCE` and others.